packages/docs-reanimated/src/components/InteractivePlayground/useEnteringExitingAnimationPlayground/Example.tsx

```diff
@@ -0,0 +1,270 @@
+import React from 'react';
+import { View, Button, StyleSheet } from 'react-native';
+import Animated, {
+  FadeIn,
+  FadeOut,
+  FadeInLeft,
+  FadeInRight,
+  FadeInUp,
+  FadeInDown,
+  FadeOutLeft,
+  FadeOutRight,
+  FadeOutUp,
+  FadeOutDown,
+  FlipInEasyY,
+  FlipInXDown,
+  FlipInEasyX,
+  FlipInXUp,
+  FlipInYLeft,
+  FlipInYRight,
+  LightSpeedInLeft,
+  LightSpeedInRight,
+  PinwheelIn,
+  RollInRight,
+  RollInLeft,
+  RotateInDownLeft,
+  RotateInDownRight,
+  RotateInUpLeft,
+  RotateInUpRight,
+  FlipOutEasyX,
+  FlipOutEasyY,
+  FlipOutXDown,
+  FlipOutXUp,
+  FlipOutYLeft,
+  FlipOutYRight,
+  LightSpeedOutRight,
+  LightSpeedOutLeft,
+  PinwheelOut,
+  RollOutRight,
+  RollOutLeft,
+  RotateOutDownLeft,
+  RotateOutDownRight,
+  RotateOutUpLeft,
+  RotateOutUpRight,
+  SlideInRight,
+  SlideInLeft,
+  SlideInUp,
+  SlideInDown,
+  SlideOutLeft,
+  SlideOutRight,
+  SlideOutUp,
+  SlideOutDown,
+  StretchInX,
+  StretchInY,
+  StretchOutX,
+  StretchOutY,
+  BounceOut,
+  BounceOutRight,
+  BounceOutLeft,
+  BounceOutUp,
+  BounceOutDown,
+  BounceIn,
+  BounceInRight,
+  BounceInLeft,
+  BounceInUp,
+  BounceInDown,
+} from 'react-native-reanimated';
+import { EnteringExitingConfigProps } from '.';
+
+interface ExampleProps {
+  entering: EnteringExitingConfigProps;
+  exiting: EnteringExitingConfigProps;
+}
+
+export const ENTERING_ANIMATIONS = {
```

Comment:
Such split seems like it should be done in Reanimated itself, but I don't exactly know how and when. Thoughts @piaskowyk @bartlomiejbloniarz?

Reply:
💀